src/screens/Forgot/index.js

```diff
@@ -0,0 +1,88 @@
+import React, {Component} from 'react';
+import {Block, TextView, Input, Button} from '../../components';
+import {theme} from '../../constants';
+import {
+  KeyboardAvoidingView,
+  Keyboard,
+  Alert,
+  ActivityIndicator,
+} from 'react-native';
+import s from './styles';
```

Comment:
Should name is styles

Reply:
name styles in file

Reply:
yes but you should define the clearly name on import. `import styles from ...` so that we will using the `styles.` instead of `s.`